apps/src/sites/studio/pages/layouts/application.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery';
+
+window.jQuery = window.$ = $;
+
+require('jquery-ujs');
+require('jquery-ui');
+
+require('bootstrap');
+require('jquery-mask-plugin');
+require('jquery-placeholder');
+require('qtip2');
+require('jquery-query-object');
+require('timeago');
+require('selectize');
+
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/add2home.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/jquery.simulate.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/jquery.ui.touch-punch.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/jquery.overlaps.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/md5.js');
+require('script-loader!@cdo/shared/js/details-polyfill/jquery.details.js');
+require('script-loader!@cdo/shared/js/details-polyfill/details-polyfill.js');
```

Comment:
Have we validated that all these things work (i.e. I wouldn't be too confidence of test coverage). add2home in particular we've broken a few times in the past without realizing